### PR TITLE
Lisaje

### PR DESCRIPTION
**Pull request by lje1234**
Whoops this got buried under AICHE and thanksgiving. How is 2nd time around?

### CODE REVIEW

File: members/lisa_je.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+<!-- Replace `example_student` with your name -->
+<img src="/website/assets/img/lisa_je.png" alt="Placeholder Image" class="center" style="max-width: 100%">
+
+<!-- Replace `Example Student` with your name and include your start date-->
+# **Lisa Je (2023 - present)**
+
+<!-- Choose your title -- feel free to be professionally silly -->
+**Role in the Cersonsky Lab**: Collaborator
```
Comment (rosecers) [inline members/lisa_je.md:12]:
```suggestion
**Role in the Cersonsky Lab**: Unofficial Senior Grad Student
```

File: members.txt [hunk 2]
```diff
@@ -20,5 +20,5 @@ undergraduate researchers
 visitors and collaborators
  - Jigyasa Nigam 
  - Leonard and Matilda
-
+ - Lisa Je
```
Comment (rosecers) [inline members.txt:23]: Can you put yourself alphabetically? We can leave L&M at the end

**Comment by rosecers** [comment]
Looking really good! Should we close #21 ?

File: members.txt (hunk 2 above)
Comment (lje1234) [reply members.txt]: Done and committed again

File: members/lisa_je.md (hunk 1 above)
Comment (lje1234) [reply members/lisa_je.md:12]: I'm going to keep it as collaborator to be "politically correct" amongst groups (and also I committed already)

File: members/lisa_je.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+---
+
+<!-- Replace `example_student` with your name -->
+<img src="/website/assets/img/lisa_je.png" alt="Placeholder Image" class="center" style="max-width: 100%">
```
Comment (rosecers) [inline members/lisa_je.md:6]:
```suggestion
<img src="/website/assets/img/lisa_je.jpg" alt="Placeholder Image" class="center" style="max-width: 100%">
```

File: members/lisa_je.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+**Cool stuff I've done**: Adopted a dog, National Science Foundation Graduate Research Fellow (NSF GRFP), National Academy of Sciences Ford Foundation Honorable Mention
+
+
+<!-- If you have photos you would like to exhibit,
+     save them as `/assets/member_images/your_name_photo_#.png`
+     and replace example_student below -->
+
+|      |      |      |
+|:----:|:----:|:----:|
+|![](/website/assets/img/lisa_je_1.png) | ![](/website/assets/img/lisa_je_2.png) | ![](/website/assets/img/lisa_je_3.png) |
```
Comment (rosecers) [inline members/lisa_je.md:29]:
```suggestion
|![](/website/assets/img/lisa_je_1.jpg) | ![](/website/assets/img/lisa_je_2.jpg) | ![](/website/assets/img/lisa_je_3.jpg) |
```

**Review by arthur-lin1027 — COMMENTED** [review]
need to build it over in the office but after you fix this minor capitalization typo things should look good.

**Comment by rosecers** [comment]
@lje1234 and @arthur-lin1027 where is this in the review process?

**Comment by lje1234** [comment]
Because I did this way in the past and didn't approve it for a while, the website has changed with some edits that Arthur and I decided it would be easier if I just wiped this version and redo the process again. Sorry, haven't done that yet. So I will close this (and not publish it) and then submit a new pull request with the newly updated website.

**Pull request closed without merging**